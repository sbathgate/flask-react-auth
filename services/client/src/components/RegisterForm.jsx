import React from "react";

const RegisterForm = () => {
  return (
    <div>
      <h1 className="title is-1">Register</h1>
      <hr />
      <br />
    </div>
  );
};

export default RegisterForm;
