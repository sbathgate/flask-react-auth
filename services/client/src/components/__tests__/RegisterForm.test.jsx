import React from "react";
import { render, cleanup } from "@testing-library/react";

import RegisterForm from "../RegisterForm";

afterEach(cleanup);

it("renders properly", () => {
  const { getByText } = render(<RegisterForm />);
  expect(getByText("Register")).toHaveClass("title");
});

it("renders", () => {
  const { asFragment } = renderWithRouter(<RegisterForm />);
  expect(asFragment()).toMatchSnapshot();
});
