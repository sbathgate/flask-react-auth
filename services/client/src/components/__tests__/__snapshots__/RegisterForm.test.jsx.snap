// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div>
    <h1
      class="title is-1"
    >
      Register
    </h1>
    <hr />
    <br />
  </div>
</DocumentFragment>
`;
